recipes/libsndfile/all/conandata.yml

```diff
@@ -2,3 +2,10 @@ sources:
   "1.0.29":
     url: "https://github.com/erikd/libsndfile/archive/v1.0.29.tar.gz"
     sha256: "c5541ccfa672b3f461ac21a38cdd37b1fe061cc62a422800b9d53649dee549ee"
+  "1.0.30":
+    url: "https://github.com/erikd/libsndfile/archive/v1.0.30.tar.gz"
+    sha256: "5942b963d1db3ed8ab1ffb85708322aa9637df76d9fe84e1dfe49a97a90e8f47"
+patches:
+  "1.0.30":
+    - base_path: source_subfolder
+      patch_file: patches/0001-turn-to-warning.patch
```

Comment:
you can change patch name to something like `0001-disable-static-runtime-logic.patch`. For MinGW, `1.0.29` also requires similar patch.

Reply:
I've asked to upstream if static link with MinGW was required: https://github.com/libsndfile/libsndfile/pull/538#discussion_r502784633

Reply:
Thanks!